snmp/metadata.csv

```diff
@@ -1,4 +1,5 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+snmp.devices_monitored,gauge,,,,Devices monitored count. Make 'sum by {X}' queries to count all the Devices with the tag X.,0,snmp,
```

Comment:
To be precise a `sum:snmp.devices_monitored{X} by {Y}` will count the total number of devices with the tag X for each value of Y.

Reply:
Yes, agree, but in this case `value` is always 1. I stole wording from vSphere :D